tests/onnx/data/reference_graphs/original_nncf_graph/densenet121.dot

```diff
@@ -42,343 +42,337 @@ strict digraph  {
-"341 /features/denseblock4/denselayer11/conv2/Conv" [id=341, type=Conv];
-"342 /features/denseblock4/denselayer12/Concat" [id=342, type=Concat];
-"343 /features/denseblock4/denselayer12/norm1/BatchNormalization" [id=343, type=BatchNormalization];
-"344 /features/denseblock4/denselayer12/relu1/Relu" [id=344, type=Relu];
-"345 /features/denseblock4/denselayer12/conv1/Conv" [id=345, type=Conv];
-"346 /features/denseblock4/denselayer12/relu2/Relu" [id=346, type=Relu];
-"347 /features/denseblock4/denselayer12/conv2/Conv" [id=347, type=Conv];
-"348 /features/denseblock4/denselayer13/Concat" [id=348, type=Concat];
-"349 /features/denseblock4/denselayer13/norm1/BatchNormalization" [id=349, type=BatchNormalization];
-"350 /features/denseblock4/denselayer13/relu1/Relu" [id=350, type=Relu];
-"351 /features/denseblock4/denselayer13/conv1/Conv" [id=351, type=Conv];
-"352 /features/denseblock4/denselayer13/relu2/Relu" [id=352, type=Relu];
-"353 /features/denseblock4/denselayer13/conv2/Conv" [id=353, type=Conv];
-"354 /features/denseblock4/denselayer14/Concat" [id=354, type=Concat];
-"355 /features/denseblock4/denselayer14/norm1/BatchNormalization" [id=355, type=BatchNormalization];
-"356 /features/denseblock4/denselayer14/relu1/Relu" [id=356, type=Relu];
-"357 /features/denseblock4/denselayer14/conv1/Conv" [id=357, type=Conv];
-"358 /features/denseblock4/denselayer14/relu2/Relu" [id=358, type=Relu];
-"359 /features/denseblock4/denselayer14/conv2/Conv" [id=359, type=Conv];
-"360 /features/denseblock4/denselayer15/Concat" [id=360, type=Concat];
-"361 /features/denseblock4/denselayer15/norm1/BatchNormalization" [id=361, type=BatchNormalization];
-"362 /features/denseblock4/denselayer15/relu1/Relu" [id=362, type=Relu];
-"363 /features/denseblock4/denselayer15/conv1/Conv" [id=363, type=Conv];
-"364 /features/denseblock4/denselayer15/relu2/Relu" [id=364, type=Relu];
-"365 /features/denseblock4/denselayer15/conv2/Conv" [id=365, type=Conv];
-"366 /features/denseblock4/denselayer16/Concat" [id=366, type=Concat];
-"367 /features/denseblock4/denselayer16/norm1/BatchNormalization" [id=367, type=BatchNormalization];
-"368 /features/denseblock4/denselayer16/relu1/Relu" [id=368, type=Relu];
-"369 /features/denseblock4/denselayer16/conv1/Conv" [id=369, type=Conv];
-"370 /features/denseblock4/denselayer16/relu2/Relu" [id=370, type=Relu];
-"371 /features/denseblock4/denselayer16/conv2/Conv" [id=371, type=Conv];
-"372 /features/denseblock4/Concat" [id=372, type=Concat];
-"373 /features/norm5/BatchNormalization" [id=373, type=BatchNormalization];
-"374 /Relu" [id=374, type=Relu];
-"375 /GlobalAveragePool" [id=375, type=GlobalAveragePool];
-"376 /Flatten" [id=376, type=Flatten];
-"377 /classifier/Gemm" [id=377, type=Gemm];
-"378 nncf_model_input_0" [id=378, type=nncf_model_input];
-"379 nncf_model_output_0" [id=379, type=nncf_model_output];
+"43 /features/transition1/pool/AveragePool" [id=43, type=AveragePool];
```

Comment:
Is there any understanding of the consequences of these architectural changes?
Do the patterns remain the same? What about the Quantizer's placement? What about the accuracy?

Reply:
I took a look at the new FQ placements and it looks good! We didn't validate the metrics in these tests

Reply:
But we have E2E for ONNX. Have you looked at it? Or they contains a different Torch version as a requirement?

Reply:
I ran a job and waiting for the results - job num 546.
But for E2E we do not use torch and torchvision, but I run to be sure that there is no problem with removing `Jinga2`